### PR TITLE
[Test] e2e plugin for gke

### PR DESCRIPTION
**Pull request by cheina97**
# Description

This PR adds the GKE provider to the E2E tests.

### CODE REVIEW

File: test/e2e/pipeline/infra/gke/setup.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    wait "$PID"
+done
+
+for i in $(seq 1 "${CLUSTER_NUMBER}");
+do
+  # install local-path storage class
+  install_local_path_storage "${TMPDIR}/kubeconfigs/liqo_kubeconf_${i}"
+
+  # Install kyverno
+  install_kyverno "${TMPDIR}/kubeconfigs/liqo_kubeconf_${i}"
```
Comment (fra98) [inline test/e2e/pipeline/infra/gke/setup.sh:101]: This step is not necessary anymore and can be removed as moved to CI in #2708

File: test/e2e/pipeline/infra/gke/setup.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  GKE_CLUSTER_ZONE=${GKE_ZONES[$i-1]}
+
+  gke_generate_kubeconfig "${GKE_CLUSTER_ID}" "${GKE_CLUSTER_ZONE}" "${TMPDIR}/kubeconfigs/liqo_kubeconf_${i}"
+  PIDS+=($!)
+done
+
+# Generate GKE kubeconfigs
+for PID in "${PIDS[@]}"; do
+    wait "$PID"
+done
```
Comment (fra98) [inline test/e2e/pipeline/infra/gke/setup.sh:115]: Isn't it risky to parallelize this? Consider that the gcloud function relies on an exported variable to store the kubeconfig

**Review by fra98 — APPROVED** [review]

File: test/e2e/pipeline/infra/gke/setup.sh (hunk 2 above)
Comment (cheina97) [reply test/e2e/pipeline/infra/gke/setup.sh:115]: It's not parallelized, but i forgot to remove the wait

**Comment by cheina97** [comment]
-rebase testìtrue

**Review by fra98 — APPROVED** [review]

**Pull request merged**